docs/snippets/ov_preprocessing_migration.c

```diff
@@ -145,11 +145,3 @@ int main_new() {
     return 0;
 }
 
-int main_old() {
```

Comment:
Looks like all `migration` files may be removed

@ilya-lavrenov please, comment. If yes, let's remove it in next PR

Reply:
yes, they should be removed